ember/app/components/project-form/component.js

```diff
@@ -0,0 +1,101 @@
+import { action } from '@ember/object';
+import { scheduleOnce } from '@ember/runloop';
+import { service } from '@ember/service';
+import Component from '@glimmer/component';
+import emptyChangeset from 'outdated/utils/empty-changeset';
+import ProjectValidations from 'outdated/validations/project';
+import { tracked } from 'tracked-built-ins';
+
+export default class ProjectFormComponent extends Component {
+  // Services
```

Comment:
That's obvious :wink: Such comments are obsolete as they don't really have any added value

Reply:
Removed here: https://github.com/adfinis/Outdated/pull/29/commits/c965aee52e37ef22d4143f91c193b9f67274973a#diff-b2fa88cc6fbb01d460606fa429ca33de8fadb4914a6c0e04b60fd8c6297fb999